### PR TITLE
Created tables and relations

### PR DESCRIPTION
**Pull request by MScott1118**

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -6,24 +6,119 @@ generator client {
...
+
+model Preferences {
+    id Int @id @default(autoincrement())
+    user Users @relation(fields:[user_id], references: [user_id])
+    user_id Int @unique
+    preference_genre String
+    genre_weighting Int
+    preference_release_date String
+    release_date_weighting Int
+    preference_imdv_rating Float
```
Comment (HaylzRandom) [inline prisma/schema.prisma:51]: typo in preference_imdv_rating

File: prisma/schema.prisma [hunk 2]
```diff
@@ -6,24 +6,119 @@ generator client {
...
-model Post {
-    id        Int      @id @default(autoincrement())
-    name      String
+model Movies {
+    id Int @id @default(autoincrement())
+    movie_title String
+    genre MoviesOnGenres[]
+    overview String
+    release_date String
+    idmb_rating Float
```
Comment (chiara2201) [inline prisma/schema.prisma:23]: imdb misspelt is the only thing I can think of changing :D

File: prisma/schema.prisma [hunk 3]
```diff
@@ -6,24 +6,119 @@ generator client {
...
+    vote_count Int
+    poster_path String
+    runtime Int
+    trailer String
+}
+
```
Comment (HaylzRandom) [inline prisma/schema.prisma:29]: Are we having a is_featured column for showing off featured movies?

**Review by HaylzRandom — CHANGES_REQUESTED** [review]
Added some comments regarding the tables, few typos and questions!

**Review by RGBlife — APPROVED** [review]

**Review by HaylzRandom — APPROVED** [review]
All looking good with the changes

**Pull request merged**